tests/backend/Functions.Tests/AnalyzeTests.cs

```diff
@@ -37,10 +41,14 @@ public class AnalyzeTests
             request.Content.Headers.Add("Content-Encoding", "gzip");
             request.Content.Headers.Add("Content-Type", "application/json");
 
-            var response = await Analyze.Run(request, new DoNothingTraceWriter());
+            var workflowQueue = Substitute.For<ICollector<WorkflowQueueMessage>>();
+            var response = await Analyze.Run(request, workflowQueue, NullLogger.Instance);
             var body = await response.Content.ReadAsStringAsync();
 
-            Assert.True(Guid.TryParse(body, out var _));
+            Guid submissionId;
+            Assert.True(Guid.TryParse(body, out submissionId));
```

Comment:
you can now do `Assert.True(Guid.TryParse(body, out var submissionId))`. However, since you don't need the result, you can do `Assert.True(Guid.TryParse(body, out _))` and the compiler will even do some optimizations (not important for testing, but potentially in production code in certain circumstances)